content/engineering/our-approach/release-checklist.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Release checklist
+sidenav: true
+sticky_sidenav: true
+tags: engineering
+permalink: /engineering/our-approach/release-checklist/
+redirect_from:
+  - engineering/release-checklist/
+layout: layouts/page
+eleventyNavigation:
+  parent: engineering_approach
+  key: Release checklist
+  order: 10
+  title: Release checklist
+subnav:
+  - text: Introducing release checklist
+    href: "#introducing-release-checklist"
+  - text: What should it look like?
+    href: "#what-should-it-look-like"
+  - text: Questions to consider
+    href: "#questions-to-consider"
+  - text: Resources
+    href: "#resources"
+---
+## Introducing release checklist
+
+The release checklist is a list of all tasks that need to be completed during the release process -- be it a launch or updating software. The checklist helps ensure that all tasks are completed prior to a release, reducing risk, and helps with following up on what needs to be done once release is complete.  When people see all the tasks listed and who owns each item, it increases confidence that the release will be successful. 
```

Comment:
```suggestion
The release checklist is a list of tasks that need to be completed during a major release, like an initial site launch. 

The checklist helps ensure that all tasks are completed prior to a release, reducing risk, and helps with following up on what needs to be done once release is complete.  When people see all the tasks listed and who owns each item, it increases confidence that the release will be successful. 
```

Reply:
@kewlguy781 -- does this do a good job of capturing our discussion? 

@hbillings -- does this do a good job of answering your question about what `release` means in this context?

Reply:
This looks great, thank you both so much!